src/libraries/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFileExtensions.ZipArchive.Create.cs

```diff
@@ -5,74 +5,73 @@
+        /// ## Examples
+        /// The following example shows how to create a new entry in a zip archive from an existing file.
+        /// [!code-csharp[System.IO.Compression.ZipArchive#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.io.compression.ziparchive/cs/program3.cs#3)]
+        /// [!code-vb[System.IO.Compression.ZipArchive#3](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.io.compression.ziparchive/vb/program3.vb#3)]
+        /// ]]></format></remarks>
+        /// <exception cref="System.ArgumentException"><paramref name="sourceFileName" /> is <see cref="string.Empty" />, contains only white space, or contains at least one invalid character.
+        /// -or-
+        /// <paramref name="entryName" /> is <see cref="string.Empty" />.</exception>
+        /// <exception cref="System.ArgumentNullException"><paramref name="sourceFileName" /> or <paramref name="entryName" /> is <see langword="null" />.</exception>
+        /// <exception cref="System.IO.PathTooLongException">In <paramref name="sourceFileName" />, the specified path, file name, or both exceed the system-defined maximum length.</exception>
+        /// <exception cref="System.IO.DirectoryNotFoundException"><paramref name="sourceFileName" /> is invalid (for example, it is on an unmapped drive).</exception>
+        /// <exception cref="System.IO.IOException">The file specified by <paramref name="sourceFileName" /> cannot be opened, or is too large to be updated (current limit is <see cref="int.MaxValue" />).</exception>
+        /// <exception cref="System.UnauthorizedAccessException"><paramref name="sourceFileName" /> specifies a directory.
+        /// -or-
+        /// The caller does not have the required permission to access the file specified by <paramref name="sourceFileName" />.</exception>
+        /// <exception cref="System.IO.FileNotFoundException">The file specified by <paramref name="sourceFileName" /> is not found.</exception>
+        /// <exception cref="System.NotSupportedException">The <paramref name="sourceFileName" /> parameter is in an invalid format.
+        /// -or-
+        /// The zip archive does not support writing.</exception>
+        /// <exception cref="System.ObjectDisposedException">The zip archive has been disposed.</exception>
         public static ZipArchiveEntry CreateEntryFromFile(this ZipArchive destination, string sourceFileName, string entryName) =>
             DoCreateEntryFromFile(destination, sourceFileName, entryName, null);
 
-
-        /// <summary>
-        /// <p>Adds a file from the file system to the archive under the specified entry name.
-        /// The new entry in the archive will contain the contents of the file.
-        /// The last write time of the archive entry is set to the last write time of the file on the file system.
-        /// If an entry with the specified name already exists in the archive, a second entry will be created that has an identical name.
```

Comment:
Is it okay to lose this info?

Reply:
Good point. I think _some_ of this information should be part of the official documentation. Thanks for catching.